src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -231,6 +231,20 @@ export class AccountManagementService implements IAccountManagementService {
 		return this._accountStore.getAllAccounts();
 	}
 
+	public async getDstsToken(serverName, databaseName): Promise<string> {
+		let accounts = await this.getAccountsForProvider('dstsAuth');
```

Comment:
I think this warrants some more discussion - I don't like having a function specifically to get this type of token. What happens when we want to add another security token type?

I think it'd be better if we looked into extending the current getAccountSecurityToken to take in a "context" or something - basically a property bag of extra stuff (think headers for HTTP requests) that the provider uses when fetching the actual token.

Reply:
Makes sense. I'll look into it.